r/R/python.R

```diff
@@ -315,3 +269,11 @@ pyarrow_version <- function() {
 }
 
 pyarrow_version_pointers_changed <- "7.0.0"
+
+backwards_compatible_pointer_double <- function(ptr) {
```

Comment:
Thanks. Which one do you think it might be better? `pyarrow_compatible_pointer()` or `backwards_compatible_pointer()`? I'd be in favour of spelling out pointer since it might make it a bit more readable.

Reply:
I would vote for `pyarrow_compatible_pointer()`, since pyarrow is what the compatibility layer is for here

Reply:
That's the name I went with (in the [0778f50](https://github.com/apache/arrow/pull/12168/commits/0778f5074e336e09138b924b641ef02900f9b4bf) commit)

Reply:
I'll mark this as resolved then, unless anyone objects